gpt_index/langchain_helpers/chain_wrapper.py

```diff
@@ -41,6 +45,19 @@ def _get_llm_metadata(llm: BaseLanguageModel) -> LLMMetadata:
             max_input_size=llm.modelname_to_contextsize(llm.model_name),
             num_output=llm.max_tokens,
         )
+    elif isinstance(llm, ChatOpenAI):
+        if llm.model_name == 'gpt-4':
```

Comment:
is this integrated into langchain now?

Reply:
Let me check latest, and revise this PR.

Reply:
Nope, not available yet as of 0.0.115.

Reply:
huh